docs/Documentation/OutputVarList.md

```diff
@@ -3,176 +3,180 @@
+| OUT_LAKE_BF_IN         	| ncoming baseflow from lake catchment                            	                            | mm  	        |
+| OUT_LAKE_BF_IN_V       	| ncoming volumetric baseflow from lake catchment                 	                            | m<sup>3</sup> |
+| OUT_LAKE_BF_OUT        	| utgoing baseflow from lake to channel network                  	                            | mm  	        |
+| OUT_LAKE_BF_OUT_V      	| utgoing volumetric baseflow from lake to channel network        	                            | m<sup>3</sup> |
+| OUT_LAKE_CHANNEL_IN    	| hannel inflow from upstream                                    	                            | mm  	        |
+| OUT_LAKE_CHANNEL_IN_V  	| olumetric channel inflow from upstream                          	                            | m<sup>3</sup> |
+| OUT_LAKE_CHANNEL_OUT   	| hannel outflow from lake to channel network                     	                            | mm  	        |
+| OUT_LAKE_CHANNEL_OUT_V 	| olumetric channel outflow from lake to channel network          	                            | m<sup>3</sup> |
+| OUT_LAKE_DSTOR         	| hange in lake moisture storage (liquid plus ice cover)         	                            | mm  	        |
+| OUT_LAKE_DSTOR_V       	| olumetric change in lake moisture storage (liquid plus ice cover)                             | m<sup>3</sup> |
+| OUT_LAKE_DSWE          	| hange in swe on top of lake ice                                	                            | mm  	        |
+| OUT_LAKE_DSWE_V        	| olumetric change in swe on top of lake ice                      	                            | m<sup>3</sup> |
+| OUT_LAKE_EVAP          	| et evaporation from lake surface                                	                            | mm  	        |
+| OUT_LAKE_EVAP_V        	| et volumetric evaporation from lake surface                     	                            | m<sup>3</sup> |
+| OUT_LAKE_PREC_V        	| olumetric precipitation over lake surface                       	                            | m<sup>3</sup> |
+| OUT_LAKE_RCHRG         	| echarge from lake to surrounding wetland                        	                            | mm  	        |
+| OUT_LAKE_RCHRG_V       	| olumetric recharge from lake to surrounding wetland             	                            | m<sup>3</sup> |
+| OUT_LAKE_RO_IN         	| ncoming runoff from lake catchment                             	                            | mm  	        |
+| OUT_LAKE_RO_IN_V       	| ncoming volumetric runoff from lake catchment                   	                            | m<sup>3</sup> |
+| OUT_LAKE_VAPFLX        	| utgoing sublimation from snow on top of lake ice                	                            | mm  	        |
+| OUT_LAKE_VAPFLX_V      	| utgoing volumetric sublimation from snow on top of lake ice     	                            | m<sup>3</sup> |
```

Comment:
Lines 50-70 have the first letter cut off of the description of the output variables